.github/workflows/release_checks.yml

```diff
@@ -0,0 +1,19 @@
+name: Release Checks
+on:
+  push:
+    branches:
+      - 'release-v[0-9]+\.[0-9]+\.[0-9]+(\-[0-9]+)?'
+  pull_request:
+    branches:
+      - 'release-v[0-9]+\.[0-9]+\.[0-9]+(\-[0-9]+)?'
+    tags:
+      - 'v[0-9]+\.[0-9]+\.[0-9]+(\-[0-9]+)?'
+jobs:
+  release_label:
+    name: Release Label Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: docker://agilepathway/pull-request-label-checker:latest
+        with:
+          all_of: L-release
```

Comment:
are you sure we want a release label? i though a release PR will just get `L-skip` to not show in the Changelog.

Also - i just realize that a `L-skip` label doesn't seem to exist ⭕

Reply:
Why do we need a L-skip when if you dont have a Label, the PR will be skipped
And I got the L-release from #661 the TODO

Reply:
if you check for `one-of: L-` the check will fail if no `L-` tag is present. By design, to remind people to add the label.

So if you intentionally want no changelog entry, a `L-skip` is needed

Reply:
I will add it back